1671816954705	Sync.LogManager	DEBUG	Flushing file log
1671816954764	Sync.LogManager	DEBUG	Log cleanup threshold time: 1670952954764
1671816954793	Sync.LogManager	DEBUG	Done deleting files.
1671820554691	Sync.Service	DEBUG	User-Agent: Firefox/107.0.1 (Linux x86_64) FxSync/1.109.0.20221128185858.desktop
1671820554691	Sync.Service	INFO	Starting sync at 2022-12-23 19:35:54 in browser session h94MjSH3NDSb
1671820554691	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1671820554695	Sync.Status	INFO	Resetting Status.
1671820554695	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1671820554695	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1671820554695	FirefoxAccounts	DEBUG	getOAuthToken enter
1671820554696	FirefoxAccounts	TRACE	getCachedToken returning cached token
1671820554696	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1671820554696	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1671820554697	Sync.SyncAuthManager	INFO	Getting sync key
1671820554698	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1671820554698	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1671820554698	FirefoxAccounts	DEBUG	getOAuthToken enter
1671820554698	FirefoxAccounts	TRACE	getCachedToken returning cached token
1671820554698	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1671820554698	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1671820554699	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1671820554703	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1671820554703	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 491\"  data: no]"}
1671820554704	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1671820554704	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:41:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:64:16
_tokenServerExchangeRequest@tokenserverclient.js:243:13
1671820554704	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1671820554704	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1671820554704	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:41:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:64:16
_tokenServerExchangeRequest@tokenserverclient.js:243:13
1671820554706	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1671820554708	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/201416140/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1671820554708	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1671820554708	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1671820554708	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1671820554708	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1671820554709	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1671820554713	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
